src/ensembl/src/content/app/browser/Browser.tsx

```diff
@@ -65,6 +66,22 @@ import { EnsObject } from 'src/shared/state/ens-object/ensObjectTypes';
 
 import styles from './Browser.scss';
 
+const client = new ApolloClient({
+  uri: '/thoas',
+  cache: new InMemoryCache({
+    typePolicies: {
+      Gene: {
+        keyFields: ['stable_id'],
+        fields: {
+          slice: {
+            merge: false
+          }
+        }
+      }
+    }
+  })
+});
```

Comment:
After our conversation with @imransl last code review, we decided to move the code for the instantiation of Apollo client into its own file, so that it can be initialized only once and can be reused outside of React components. Imran is doing it [here](https://github.com/Ensembl/ensembl-client/pull/437/files#diff-3cae4751bd482c8e57ada5e39c83ae417e731ed82235b1b2b59190b187ca30fbR1). This is something to be aware of and later reuse in genome browser.

Reply:
I have removed the graphql related changes from this PR as discussed.